### DISCUSSION TITLE
Welcome to MetaOutput Discussions!

### DISCUSSION THREAD

**Discussion by viacheslav-lozinskyi** [viacheslav-lozinskyi/MetaOutput#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by teodorivanov1** [comment]
Congrats for the hard work. I wish pressing Shift + F12 would show all references.

**Reply by viacheslav-lozinskyi** [reply]
I can't understand your request.
Where should be implemented Shift - F12, and which references do you want to see?
